.golangci.yml

```diff
@@ -11,8 +11,6 @@ linters-settings:
       - '^exec\.Cmd$'
       - '^exec\.Command$'
       - '^exec\.CommandContext$'
-      # This should only be used by github.com/bufbuild/buf-language-server
-      - '^bufcli\.NewModuleReaderAndCreateCacheDirsWithExternalPaths'
```

Comment:
The LSP will need to be updated to support new cache paths.

Reply:
The LSP was just a prototype and isn't being actively developed, we don't need to update it